pydicom/values.py

```diff
@@ -192,26 +262,32 @@ def convert_PN(byte_string, is_little_endian, struct_format=None, encoding=None)
 
     # We would like to return string literals
     if not in_py2:
-        valtype = lambda x: PersonName(x, *args).decode()
+        valtype = lambda x: PersonName(x, *args).decode()  # nopep8
     else:
-        valtype = lambda x: PersonName(x, *args)
+        valtype = lambda x: PersonName(x, *args)  # nopep8
 
```

Comment:
I just tried it, and messed it up, too. As I said - I find the lambda much more readable...